static/docs/commands-reference/update.md

```diff
@@ -0,0 +1,30 @@
+# update
+
+Update dependencies and reproduce specified DVC-files.
```

Comment:
I'm not sure we need to mention "reproduce" term here. It is implementation specific.

Reply:
Oops I forgot to write a description haha sorry. This is just the `-h` output of the command. Will rewrite and lyk.

Reply:
I would say we should update the help message as well. I would not use `reproduce` term here. It does not mean anything from the user perspective. I'm not sure if dependencies is the right term here as well. From the user perspective it's just about updating the import-ed data.

Reply:
OK updated in this repo in 6637f64, what do you think?
And added command output update to my core repo PR in https://github.com/iterative/dvc/pull/2299/commits/80c30cebea07ce8b841f41dcf84a9499ce46d7bd

Reply:
Looks good!